test/Microsoft.AspNetCore.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -289,7 +289,7 @@ public void HtmlAttributes_IgnoresCase(string originalName, string updatedName)
         {
             // Arrange
             var executionContext = new TagHelperExecutionContext("p", TagMode.StartTagAndEndTag);
-            executionContext.HTMLAttributes[originalName] = "hello";
+            executionContext.AddHtmlAttribute(originalName, "hello");
```

Comment:
@rynowak FYI can see both sides of this point:
- `AddHtmlAttribute()` exists and does the same work as the proposed `Add(string, object)` method.
- `Add()` is a more obvious replacement for `attributes["name"] = value` and should be more discoverable than the `AddHtmlAttribute()` "shortcut".